src/clients.rs

```diff
@@ -266,7 +266,11 @@ impl StateClient for State {
             state.cache.insert(psk, rsp.height, note)?;
         }
 
-        println!("Last block: {}", last_height);
+        println!(
+            "Last block: {} (Epoch {})",
+            last_height,
+            last_height / EPOCH
+        );
```

Comment:
I would like to remove the epoch from here.
This "Last block" is not the current block height, but the last block at which the cache is synced with.
Hence, the epoch here is completely misleading imo

Reply:
@herr-seppia shouldn't we make users aware of the current or last cache synced epoch though somewhere?

Reply:
Indeed we should
I'm just concerned about the misleading information here.

Users can misunderstand the cache block height to the network block height

I'm for change the "Last block" label too tbh, but this is for another issue

Reply:
I wil remove the epoch display here, and than we can look at ways and places to display information in a way that is less misleading.